next.config.js

```diff
@@ -20,5 +20,12 @@ const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
   trailingSlash: true,
+  experimental: {
+    // This is experimental but can
+    // be enabled to allow parallel threads
+    // with nextjs automatic static generation
+    workerThreads: false,
```

Comment:
this bit here seems to fix all of the sporadic `Bad Gateway` errors, which I believe were the result of the build process sending too many parallel requests to Drupal